server/src/main/java/io/druid/client/BrokerServerView.java

```diff
@@ -73,6 +74,7 @@
   private final TierSelectorStrategy tierSelectorStrategy;
   private final ServiceEmitter emitter;
   private final Predicate<Pair<DruidServerMetadata, DataSegment>> segmentFilter;
+  private final AuthConfig authConfig;
```

Comment:
The auth logic is part of @ResourceFilters(StateResourceFilter.class) in BrokerResource